### DISCUSSION TITLE
Inline formulas not working for some situations

### DISCUSSION THREAD

**Discussion by arolus** [community/community#30740] [Other Feature Feedback, Questions, & Ideas]
For example, this text's formulas are displayed ok:
You can simplify all these fractions to $\frac{1}{3}$.
Você pode simplificar todas essas frações até $\frac{1}{3}$.

But this text formulas are not displayed:
You can simplify all these fractions to $\frac{1}{3}$.
Você pode simplificar todas essas frações até $\frac{1}{3}$.

It is the same text. Shown twice.
Please advise.

In case it is displayed differently on other systems, here is a screenshot:
<img width="514" alt="screen shot 2022-08-25 at 08 12 00" src="https://user-images.githubusercontent.com/77734966/186580298-9f341943-708e-4870-ad0a-0480b37e3b8c.png">

**Comment by AaronYong179** [comment]
Had the same issue, posted a separate thread before coming back again and finally stumbling upon this. Inline math seems to work once and fails afterwards. 

Here is an example copy-pasted from from the [GitHub blog post on math support](https://github.blog/2022-05-19-math-support-in-markdown/)

"
When $a \ne 0$, there are two solutions to $(ax^2 + bx + c = 0)$

and they are
$$ x = {-b \pm \sqrt{b^2-4ac} \over 2a} $$
"

The bottom line doesn't render correctly at all.

These: $a \ne 0$ and $(ax^2 + bx + c = 0)$ do not seem to work either.
